src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+            break;
+        default:
+            break;
+        }
+        return out;
+    }
+    public static void line() {
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    public static void showList(Tasks user) {
+        for (int i = 0; i < user.listLength; i++) {
+            System.out.println("\t" + (i + 1) + ". " + user.list[i] + "\n");
+        }
+    }
+    public static void addTodo(String input, Tasks user) {
+        user.list[user.listLength] = new Todo(input.substring(5));
+        user.listLength ++;
+        user.tasksIncomplete ++;
+    }
+    public static void addDeadline(String input, Tasks user) {
+        user.list[user.listLength] = new Deadline(input.substring(9, input.indexOf("/") - 1), input.substring(input.indexOf("/") + 1), input.substring(9));
+        user.listLength ++;
+        user.tasksIncomplete ++;
+    }
+    public static void addEvent(String input, Tasks user) {
+        user.list[user.listLength] = new Event(input.substring(6, input.indexOf("/") - 1), input.substring(input.indexOf("/") + 1), input.substring(6));
+        user.listLength ++;
+        user.tasksIncomplete ++;
+    }
+    public static String markDone(String input, Tasks user) throws ArrayIndexOutOfBoundsException, MultiMarkDoneException {
+        int index = Integer.parseInt(input.substring(5)) - 1;
+        if (index < user.listLength &&  index >= 0) {
+            if (user.list[index].complete) {
+                throw new MultiMarkDoneException();
+            }
+            user.list[index].markComplete();
+            user.tasksIncomplete --;
+            return ("\tAs you wish sir. I have marked this task as done:\n\t[X] " + user.list[index].item + "\n\tNow you have " + user.tasksIncomplete + " incomplete tasks.");
+        } else {
+            throw new ArrayIndexOutOfBoundsException();
+
+        }
+    }
+
+    public static void deleteEvent(String input, Tasks user) throws ArrayIndexOutOfBoundsException {
+        int index = Integer.parseInt(input) - 1;
+        if (index < user.listLength && index >= 0) {
+            if (!user.list[index].complete) {
+                user.tasksIncomplete --;
+            }
+            System.out.println("\tSir, as per your request, I have deleted the task:\n\t" + user.list[index] + "\n\tNow you have " + user.tasksIncomplete + " incomplete tasks.");
+            for (int i = index; i < user.listLength; i++) {
+                user.list[i] = user.list[i + 1];
+            }
+            user.listLength --;
+        } else {
+            throw new ArrayIndexOutOfBoundsException();
+        }
+    }
+
+    public static void response(String input, Tasks user) throws IllegalCommandException, IllegalTaskException, DueDateFormatException, MultiMarkDoneException, IOException, DeleteTaskFormatException {
+        String[] inputArr = input.split(" ");
+        switch (inputArr[0]) {
+        case ("list"):
```

Comment:
The brackets `()` for each case are unnecessary - see [switch statement style](https://se-education.org/guides/conventions/java/basic.html#layout).